src/interactive/Dropdown.jsx

```diff
@@ -227,7 +227,6 @@ class Dropdown extends React.PureComponent {
 						aria-hidden={!isActive}
 						style={{
 							left: this.state.left,
-							right: align === 'right' && noPortal && `-${maxWidth}`,
```

Comment:
Looks like maybe this was branched from something not up to date with `master`?

Reply:
Nope, that line was intentionally removed. It was causing weirdness with right-aligned modals that weren't portalled